### ISSUE TITLE
[Python] Implement new Concept Exercise: pass

### ISSUE THREAD

**Issue by DavidGerva** [exercism/v3#2040]
This issue describes how to implement the concept exercise, for the Python track, which should explain how to use the `pass` statement in python, and what it means.

## Getting started

**Please please please read the docs before starting.** Posting PRs without reading these docs will be a lot more frustrating for you during the review cycle, and exhaust Exercism's maintainers' time. So, before diving into the implementation, please read up on the following documents:

- [The features of v3](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md).
- [Rationale for v3](https://github.com/exercism/v3/blob/master/docs/rationale-for-v3.md).
- [What are concept exercise and how they are structured?](https://github.com/exercism/v3/blob/master/docs/features-of-v3.md)

Please also watch the following video:

- [The Anatomy of a Concept Exercise](https://www.youtube.com/watch?v=gkbBqd7hPrA).

## Goal

This concept exercise should convey a basic understanding of the `pass` statement in python, how to use it as a place-holder and what it does.

## Learning objectives

- use the `pass` statement to write a syntactically valid block of code

## Out of scope

- nothing

## Concepts

- the `pass` statement

## Prerequisites

- nothing

## Resources to refer to

- [The 35 words you need to python](https://yawpitchroll.com/posts/the-35-words-you-need-to-python/#pass)
- [Python pass statement official doc](https://docs.python.org/3/tutorial/controlflow.html?highlight=pass#pass-statements)

### Hints

- TBD?

### After

Nothing

## Representer

No changes required.

## Analyzer

No changes required.

## Implementing

Tests should be written using unittest.TestCase and the test file named bool_basic_test.py.

## Help

If you have any questions while implementing the exercise, please post the questions as comments in this issue.

## Edits
-

**Comment by DavidGerva** [comment]
Do we really need an exercise on 'pass'?

**Comment by BethanyG** [comment]
Good question.  @DavidGerva  -- My hot take is **no**, but then I don't know where we'd include it.  It would clutter the `basics` exercise, and it doesn't make a whole lot of sense to chuck it into an exercise say on higher-order functions or some other function related exercise.  I suppose we could shoehorn it into something on classes or a more complex exercise where we force several passes ...... 🤔   @cmccandless  - what are your thoughts on where `pass` would land, and if it needs its very own exercise??

**Comment by cmccandless** [comment]
I don't think it fits as a dedicated exercise. Don't the stubs use pass? Can we just include a brief note on what that keyword means in the basics exercise?

**Comment by DavidGerva** [comment]
> 
> 
> I don't think it fits as a dedicated exercise. Don't the stubs use pass? Can we just include a brief note on what that keyword means in the basics exercise?

I have to admit (I hold some courses within the company I work for) sometimes people struggle to understand the usefulness and meaning of `pass`. But I agree, I really cannot find an idea of exercise with the "only" use of `pass`.
So, if you agree I'll close this issue.

**Comment by cmccandless** [comment]
Closing as "will not implement"

**Comment by ynfle** [comment]
I'm not sure if these were being covered, but Abstract Base Classes use them. See https://docs.python.org/3/library/abc.html#abc.ABC
```python
from abc import ABC

class MyABC(ABC):
    pass
```
